releasewarrior/wiki_data.py

```diff
@@ -159,6 +159,7 @@ def write_and_commit(data, data_path, wiki_path, commit_msg, logger, config, wik
     logger.debug(wiki_path)
     logger.debug(corsica_path)
     commit([data_path, wiki_path, corsica_path], commit_msg, logger, config)
+    push(logger, config)
```

Comment:
Like @Callek was suggesting, if we condition this off an option or make a separate subcommand, I'm all for it.

I think this is a really nice feature and skips some manual pain.

fwiw - right now I have the following aliases that act as subcommands so I don't need to cd and type long git commands:

alias rwl='git -C /Users/jlund/devel/mozilla/repos/releasewarrior-data pull upstream master'
alias rwd='git -C /Users/jlund/devel/mozilla/repos/releasewarrior-data diff upstream/master'
alias rwp='git -C /Users/jlund/devel/mozilla/repos/releasewarrior-data push upstream master'

Reply:
@srfraser @Callek how would we like to follow up here? Condition this call off a config option, a cli option, or no condition at all?

Reply:
@lundjordan I suggested my fix by a WIP PR in #42

Reply:
I'm happy with #42, and we anc always add more options, or a 'pass all this to git and add -C' option if it gets that far

Reply:
okay closing this for now due to last two comments